app/assets/javascripts/components/my_lessons_page.es6.jsx

```diff
@@ -0,0 +1,62 @@
+class MyLessonsPage extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      filter: 'upcoming',
+    }
+  }
+
+  handleClick(filter) {
+   this.setState({ filter: filter });
+  }
+
+  renderOption(option) {
+    switch (option) {
+      case 'upcoming':
+        style = (this.state.filter == 'upcoming' ?
+                "button button--lg button--solid-orange" :
```

Comment:
People usually indent this once if you have the `);` on its own line. Up to your preference.